tests/acceptance/features/cliCommands/removeFileVersions.feature

```diff
@@ -1,10 +1,12 @@
-@env-config
+#@env-config
```

Comment:
```suggestion
@env-config
```

Reply:
HAHHA. forgot about this. i was trying something else by commenting it.